.cursor/skills/write-zenn-dialogue-article/dialogue-guide.md

```
@@ -0,0 +1,269 @@
+ここで A先輩が見せたのは、props をグループ化するアプローチです。
+
+```tsx
+// before: バラバラの props
+type Props = { userName: string; userEmail: string; userAvatar: string /* ... */ };
+
+// after: オブジェクトでまとめる
+type Props = { user: User };
+```
+````
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)